Assets/Prefab/Spawner/Enemy spawner.prefab

```diff
@@ -0,0 +1,102 @@
+%YAML 1.1
```

Comment:
this is a cube not spawner

it is used a s a spawn point

spawner is an object that controls spawning

Reply:
Generally empty game object with icon is fine as a spawn point, i don't see a need for prefab here